data/binding/mapbinding.go

```diff
@@ -192,26 +194,31 @@ type boundReflect struct {
 	val reflect.Value
 }
 
-func (b *boundReflect) get() interface{} {
-	return b.val.Interface()
+func (b *boundReflect) get() (interface{}, error) {
+	return b.val.Interface(), nil
 }
 
-func (b *boundReflect) set(val interface{}) {
+func (b *boundReflect) set(val interface{}) error {
+	// TODO catch the panic and return as error
```

Comment:
Can we get these resolved before merge?

Reply:
I forgot, sorry. Not sure but I will look into it.

Reply:
Added :)